src/services/users.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+class Users {
+  constructor() {
+    this.users = [];
+  }
+  reset() {
+    this.users = [];
+  }
+
+  getAll() {
+    return this.users;
+  }
+
+  getById(userId) {
+    const foundUser = this.users.find(user => Number(userId) === user.id);
+
+    return foundUser || null;
+  }
+
+  createUser(name) {
```

Comment:
Add validation for the 'name' parameter to check if it's not null or an empty string.

Reply:
![image](https://github.com/mate-academy/node_accounting-app-v2/assets/124785462/ec855b3f-28e7-439e-ab9c-791b9551c698)

Reply:
![image](https://github.com/mate-academy/node_accounting-app-v2/assets/124785462/d1a0dae3-234a-4496-8fdf-feb745716cba)